NodeArray.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#define SIZE 26
+
+struct Node {
+	Node *children[SIZE];
+	bool wordEnd;
+};
```

Comment:
Please, add \n in the end of file.
Why to do this? Here: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206